config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -60,7 +60,7 @@ presubmits:
     decorate: true
     branches:
     - master
-    - release-0.16
+    - release-1.0
```

Comment:
Does this configure prow to run this test before merging into either of these two branches?
Well then what about if we want to do a 0.16 patch release? It looks like we're configuring prow to no longer run these tests before merging stuff into the release-0.16 branch. Or have I got the wrong end of the stick?